components/StyledInputAmount.js

```diff
@@ -96,6 +96,12 @@ const StyledInputAmount = ({
         e.stopPropagation();
         dispatchValue(e, parseValueFromEvent(e, precision));
       }}
+      onKeyDown={e => {
+        if (e.key === ',') {
```

Comment:
This is a breaking change for locales like French where `,` is the decimal separator.See the feedback in  https://github.com/opencollective/opencollective/issues/5073#issuecomment-1005462527